Library/Formula/iojs.rb

```diff
@@ -1,4 +1,8 @@
+require "language/javascript"
+
 class Iojs < Formula
+  include Language::JS
```

Comment:
Yeh, https://github.com/bcomnes/homebrew/commit/53b7c47ba8c869ea58643e4b11479f0633674fcd makes sense to me. I think losing the scope is a feature there (stops being as coupled to the `Formula` API). @xu-cheng previously handled the `stdout` leaking case but I can't quite remember how; I'm sure he'll be able to elaborate.